l10n_it_pec_messages/model/mail_thread.py

```diff
@@ -212,3 +295,20 @@ def _message_find_partners_pec(
             if partner_ids:
                 res = partner_ids[0]
         return res
+
+    def _FindOrCreatePartnersPec(
```

Comment:
@eLBati sì ma ho comunque deciso di lasciarlo nel caso ci servisse in futuro, potremmo anche pensare di gestirlo in modo parametrico ed inserire un campo nel server pec per decidere se creare o meno il partner all'arrivo della pec. Comunque tutto il resto continuerebbe a funzionare correttamente